### ISSUE TITLE
Order last used languages at top of the list

### ISSUE THREAD

**Issue by ronilaukkarinen** [elk-zone/elk#1550]
I'm using multiple languages so it would be nice to see those languages I use at the top of the list instead of having to look for them each time I post:

![Screen-Shot-2023-02-01-09-29-13](https://user-images.githubusercontent.com/1534150/215978890-dd196d57-87ea-4d0f-8270-11db2db63b72.png)

**Comment by edimitchel** [comment]
Goog idea, as we have a setting about language known (doesn't need translation button for post which was written in those language), we could show them as first items in the list!

**Comment by edimitchel** [comment]
I suggest @tlagoda to take that issue in order.

**Comment by moeffju** [comment]
I would like to add one consideration - it would be great if the list were either in a user-defined order or static. There's nothing worse than having the items rearrange themselves frequently, as could be the case if the sorting was by usage frequency for example, for users that use multiple languages more or less equally.

**Comment by alper** [comment]
Anything would be better than the current situation, really.

I think seeding it with the languages you would like to read makes sense (because those would realistically be the only languages that you'd write in) if that's a setting that's easy to retrieve.

**Comment by moeffju** [comment]
See also #876

**Comment by niklaswolf** [comment]
I think this was solved in https://github.com/elk-zone/elk/pull/1614 and this issue can be closed.
If you configure your known languages in the settings, they get prioritized in the language picker.

**Comment by audunmb** [comment]
> I think this was solved in #1614 and this issue can be closed. If you configure your known languages in the settings, they get prioritized in the language picker.

No, that doesn't really work. There's no obvious connection between "do not translate" and the prefered languages to post in.  Though, they are likely to be the same, they are not the same. For instance, I can read Danish and Swedish, and both variations of Norwegian, but will never post in those languages. 

Also, the do no translate-list is way shorter than the list of possible languages.
